features/poller/monitors/core/blueprint-poller.xml

```diff
@@ -0,0 +1,343 @@
+	<onmsgi:reference id="jdbcMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="jdbcMonitor"/>
+	<onmsgi:reference id="jdbcQueryMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="jdbcQueryMonitor"/>
+	<onmsgi:reference id="jdbcStoredProcedureMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="jdbcStoredProcedureMonitor"/>
+	<onmsgi:reference id="jolokiaBeanMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="jolokiaBeanMonitor"/>
+	<onmsgi:reference id="jschSshMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="jschSshMonitor"/>
+	<onmsgi:reference id="ldapMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="ldapMonitor"/>
+	<onmsgi:reference id="ldapsMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="ldapsMonitor"/>
+	<onmsgi:reference id="loopMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="loopMonitor"/>
+	<onmsgi:reference id="memcachedMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="memcachedMonitor"/>
+	<onmsgi:reference id="nrpeMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="nrpeMonitor"/>
+	<onmsgi:reference id="ntpMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="ntpMonitor"/>
+	<onmsgi:reference id="pop3Monitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="pop3Monitor"/>
+	<onmsgi:reference id="smtpMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="smtpMonitor"/>
+	<onmsgi:reference id="sshMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="sshMonitor"/>
+	<onmsgi:reference id="sslCertMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="sslCertMonitor"/>
+	<onmsgi:reference id="strafePingMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="strafePingMonitor"/>
+	<onmsgi:reference id="systemExecuteMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="systemExecuteMonitor"/>
+	<onmsgi:reference id="tcpMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="tcpMonitor"/>
+	<onmsgi:reference id="trivialTimeMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="trivialTimeMonitor"/>
+	<onmsgi:reference id="webMonitor" interface="org.opennms.netmgt.poller.ServiceMonitor" bean-name="webMonitor"/>
+
+	<!-- ActiveMQ component provided by minion-core -->
+	<reference id="queuingservice" interface="org.apache.camel.Component" filter="(alias=opennms.broker)"/>
+
+	<camelContext id="pollerMonitorsCoreMinionContext" xmlns="http://camel.apache.org/schema/blueprint">
+		<propertyPlaceholder id="properties" location="blueprint:pollerMonitorsCoreMinionProperties" />
+
+		<!-- Availability Monitor -->
+		<route id="pollAvailabilityMonitor">
+			<from uri="queuingservice:Location-{{monitoringLocationName}}" />
```

Comment:
You need to change these JMS queue names to be the same as the names used inside applicationContext-pollerd.xml: `Location-{{name}}.Poller.AvailabilityMonitor`

This is how each poller's messages get sent to the correct monitor, they are on separate queues.